tests/test_middleware.py

```diff
@@ -23,3 +24,11 @@ def test_setup_fn_is_called_with_kwargs(self, mock_middleware_setup, project_id)
         rele.setup(settings, foo="bar")
         assert mock_middleware_setup.called
         assert mock_middleware_setup.call_args_list[0][-1] == {"foo": "bar"}
+
+    def test_warns_about_deprecated_hooks(self, caplog):
```

Comment:
Do you want to assert anything with the `caplog` here?

Reply:
Uups 😊